docs/Tutorials/Create-Private-Network.md

```diff
@@ -0,0 +1,214 @@
+      "comment": "private key and this comment are ignored.  In a real chain, the private key should NOT be stored",
+      "balance": "90000000000000000000000"
+    }
+  }
+}
+```
+
+!!! warning
+    Do not use the accounts in the genesis file above on mainnet or any public network except for testing.
+    
+    The private keys are displayed so the accounts are not secure. 
+
+Your private network directories and files are now: 
+
+```bash
+Private-Network/
+├── Node-1
+│   ├── Node-1-Datadir
+│   └── privateNetworkGenesis.json
+├── Node-2
+│   ├── Node-2-Datadir
+│   └── privateNetworkGenesis.json
+└── Node-3
+    ├── Node-3-Datadir
+    └── privateNetworkGenesis.json
+```
+   
+### 3. Start First Node and Get Node Public Key 
+
+To enable nodes to discover each other, a network requires one or more nodes to be bootnodes. 
+For this private network, we will use Node-1 as the bootnode. This requires starting and stopping the 
+node and obtaining the node public key for the enode URL. 
+
+In the `Node-1` directory and start the node specifying the data directory created in step 1, 
+the genesis file created in step 2, and a network ID for your private network (`123` in this example):
+
+```bash
+pantheon --genesis=privateNetworkGenesis.json --network-id 123 --datadir=Node-1-DataDir
+``` 
+
+A node private key for Node-1 is generated and written to the `key` file. The node public key is displayed: 
+
+`2019-01-01 05:03:27.430+10:00 | main | INFO  | KeyPairUtil | Generated new key 0x5ca7eca7a9b6b8128d8b9375fe2683c43023fd1504bca31f5123bd9b086155d81d0b05a08b63c7fa1027bbd9cfa338b6c54539c72b05a56c1f52f802c87ad09c and stored it to /Users/username/Private-Network/Node-3/Node-3-DataDir/key
+20`
+
+Stop the node using **Ctrl+C**. 
+
+Use the `export-pub-key` subcommand to write the node public key to the specified file (`publicKeyNode1` in this example): 
+
+```bash
+pantheon --datadir=Node-1-Datadir --genesis=privateNetworkGenesis.json export-pub-key Node-1-Datadir/publicKeyNode1
+```
+
+Your node 1 directory is now: 
+```bash
+├── Node-1
+    ├── Node-1-Datadir
+        ├── database
+        ├── key
+        ├── publicKeyNode1
+```
+ 
+The `database` directory is created when the node is started and contains the blockchain data. 
+      
+### 4. Restart First Node as Bootnode 
+
+Restart Node-1 specifying:
+
+* An empty string for the `--bootnodes` option because this is your bootnode
+* Mining is enabled and the account to which mining rewards are paid using the `--miner-enabled` and `--miner-coinbase` options
+* JSON-RPC API is enabled using the `--rpc-enabled` option. 
+
+```bash
+pantheon --datadir=Node-1-Datadir --genesis=privateNetworkGenesis.json --bootnodes="" --network-id 123 --miner-enabled --miner-coinbase fe3b557e8fb62b89f4916b721be55ceb828dbd73 --rpc-enabled      
+```
+
+!!! info
+    The miner coinbase account is one of the accounts defined in the genesis file. 
```

Comment:
I didn't add this link because we're not actually using the dev.json file here - it's got one more account than in the genesis file for this example.